addon/index.js

```diff
@@ -358,6 +359,7 @@ exports.main = function(options) {
   ToolbarButton.init(settings);
   ExperimentNotifications.init();
   SharePrompt.init(settings);
+  FirstRun.setup(options.reason, settings);
```

Comment:
I can't seem to find the issue now, but I remember you opening one about making this general module pattern cleaner and more like Django. :+1: to this and that in general